Content/ContentTypeResolver/PageSelectionResolver.php

```diff
@@ -76,12 +76,14 @@ public function resolve($data, PropertyInterface $property, string $locale, arra
             'published' => !$this->showDrafts,
         ]);
 
-        list($pagesQuery) = $this->contentQueryBuilder->build($property->getStructure()->getWebspaceKey(), [$locale]);
+        $webspaceKey = $property->getStructure() ? $property->getStructure()->getWebspaceKey() : null;
```

Comment:
Not sure about this change. We would need some reproducable here why this change is required I currently can not reproduce it.

Reply:
/cc @thomasduenser I removed currently this change unless we have something to reproduce so we can atleast merge the other fix